source/test-your-integration/index.html.md.erb

```diff
@@ -9,13 +9,28 @@ review_in: 6 months
 
 Once you’ve [integrated your service with Authorization Code Flow][integrate.authenticate], you can test your integration with GOV.UK One Login.
 
-To make sure your integration is working as you expect it to, you should create sample users to test:
+You have 2 options for testing: 
 
-* registering a new user
-* logging in an existing user
-* test you receive the expected scopes you selected when you [chose which user attributes your service can request][integrate.choose-user-attributes], for example email and phone number
-* logging a user out
+- the [GOV.UK One Login simulator](https://docs.sign-in.service.gov.uk/test-your-integration/gov-uk-one-login-simulator/), which lets you test and verify specific user information and error codes
```

Comment:
@SteveEvans-GDS the link to the simulator page here is absolute when it should be relative, when testing the pr locally with middleman it thows an http 404 error owing to the content being relocated, so suggest it is replaced with `/test-your-integration/gov-uk-one-login-simulator/`